frontend/internal-packages/agent/src/langchain/agents/pmAgent/prompts.ts

```diff
@@ -0,0 +1,93 @@
+import { ChatPromptTemplate } from '@langchain/core/prompts'
+
+// Requirements Analysis System Prompt
+const pmAnalysisSystemPrompt = `You are PM Agent, a skilled project manager who specializes in analyzing user requirements and extracting structured Business Requirements Documents (BRDs).
+Your role is to:
+1. Analyze user input and conversation history
+2. Extract clear, structured requirements
+3. Convert ambiguous expressions into specific, actionable requirements
+4. Separate multiple use cases into individual requirements
+5. Include specific screens, operations, constraints, and processing details when available
+
+Current Database Schema:
+{schema_text}
```

Comment:
Is `schema text` really necessary for analyzing requirements? I was wondering about this.
This is information for understanding the product, but perhaps it would be better to have the user upload the documentation in a less direct way.

I know there are many pros and cons to this opinion. However, I took the stance that each agent should focus on the task and that the less unnecessary context, the better the output.
I don't think there is a right answer and I would like to discuss it!

Reply:
I've reconsidered. You're right, the PM Agent's goal is requirements definition, so schema information would likely become unnecessary context for it. I believe that information should only be known by the DB Agent. I'll plan to remove it!

Reply:
Thanks for the confirmation. It's a difficult point as there are so many issues to discuss, but I thought it was a good idea to start small. Best regards!

Reply:
Yes, basePromptVariables may need to be changed, but that is fine for another time 👍🏻